packages/aws-cdk/README.md

```diff
@@ -316,6 +316,28 @@ execute.
 $ cdk deploy --no-execute --change-set-name MyChangeSetName
 ```
 
+#### Accelerated deployments
+
+You can pass the `accelerated` flag to the `deploy` command:
+
+```console
+$ cdk deploy --accelerated [StackNames]
```

Comment:
We said "accelerated" is not a good name (and IIUC SAM are not going to use it either).

What do you think about `--hotswap`. Not my idea, but I think it exactly represents what we want and sends a message that this might not be the best thing to use in production.

Now since we are still waiting to sync up with SAM about this, let's add some experimental prefix/suffix.

So I'd go with: `--experimental-hotswap` or `--hotswap-preview` or something like that. WDTY?

Also, we decided in the RFC that this will be an enum with three options: `auto`, `only` and `never` (we can think about the names).

Reply:
> We said "accelerated" is not a good name (and IIUC SAM are not going to use it either).

Side note: I have _no idea_ what "IIUC" means. I Googled it, and got the "International Islamic University Chittagong" in Bangladesh, which I don't think is what you meant here 😃.

> What do you think about `--hotswap`. Not my idea, but I think it exactly represents what we want and sends a message that this might not be the best thing to use in production.

I don't hate `--hotswap`. Changed to use that name.

> Now since we are still waiting to sync up with SAM about this, let's add some experimental prefix/suffix.
> 
> So I'd go with: `--experimental-hotswap` or `--hotswap-preview` or something like that. WDTY?
> 

Now this I _do_ hate 😜. This makes the CLI ergonomics just too bad.

I left it as `--hotswap`, but added a note to the ReadMe that we still consider this command experimental, and that we might make breaking changes to it in the future.

> Also, we decided in the RFC that this will be an enum with three options: `auto`, `only` and `never` (we can think about the names).

Actually, it has 4 possible values:  `auto`, `only`, `never` and `ask`. Considering the complexities of implementing them, and the fact that adding an optional argument to this switch is a backwards-compatible change, I decided to leave it out from this already quite big and complex PR.

Reply:
OK, fair enough.